### PR TITLE
Add hotel precheck confirm

### PR DESCRIPTION
**Pull request by jesusfreak3**

### CODE REVIEW

File: app/controllers/precheck/confirmation_controller.rb [hunk 1]
```diff
@@ -2,6 +2,9 @@ class Precheck::ConfirmationController < PrecheckController
   def create
     redirect_to precheck_status_path and return unless Precheck::EligibilityService.new(family: @family).call
 
+    @hotel = params['hotel'] === 'Other' ? " OTHER - #{params['other_hotel']} " : params['hotel']
```
Comment (knutsenm) [inline app/controllers/precheck/confirmation_controller.rb:5]: Is the triple-equal really needed here? Is `@hotel` needed by the view?

File: app/models/family.rb [hunk 2]
```diff
@@ -86,6 +88,10 @@ def everyone_has_birthdates?
   def anyone_has_email?
     attendees.any? { |p| p.email.present? }
   end
+ 
+  def anyone_registered_for_legacy_conferences()
+    attendees.find{|a| a.conferences.find{ |c| c.name == 'Legacy'}} != nil
```
Comment (knutsenm) [inline app/models/family.rb:93]: Same comments as other PR

**Review by knutsenm — CHANGES_REQUESTED** [review]
This seems to overlap (or is a superset of?) #220

File: app/models/family.rb (hunk 2 above)
Comment (jesusfreak3) [reply app/models/family.rb:93]: Yah, i required some of the previous changes when I was working on it, so it was built based of the other Branch.

File: app/controllers/precheck/confirmation_controller.rb (hunk 1 above)
Comment (jesusfreak3) [reply app/controllers/precheck/confirmation_controller.rb:5]: Sorry I just had the stomach flu Thursday/Friday, so not thinking straight.  Good catch, hotel is not passed to the view here so will put as a local variable.  And no, its not necessary for the case quality operator... I was coding in JS/PHP dynamic language at the same time and it bled into Ruby.  Will correct these.

**Review by knutsenm — APPROVED** [review]

**Review by hlbraddock — APPROVED** [review]

**Pull request merged**